src/main/java/com/icthh/xm/ms/configuration/config/WebConfigurer.java

```diff
@@ -71,9 +76,7 @@ public void customize(WebServerFactory server) {
          * See the JHipsterProperties class and your application-*.yml configuration files
```

Comment:
Fix this java doc (serverProperties instead of JHipsterProperties)

Reply:
Done